.github/workflows/ci.yml

```diff
@@ -6,10 +6,17 @@ jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
     strategy:
       matrix:
-        version: ['1.6', '^1.7.0-rc1', 'nightly']
-        os: [ubuntu-latest, windows-latest, macOS-latest]
+        version: 
```

Comment:
Why this version is recommended?

Reply:
This form clutters less if you have a long list of versions later on. The two forms are basically the same.